web/src/beta/components/Icon/Icons/uploadSimple.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M5.375 5.12431L8 2.5L10.625 5.12431" stroke="#E0E0E0" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M8 9.5V2.50184" stroke="#E0E0E0" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M13.5 9.5V13C13.5 13.1326 13.4473 13.2598 13.3536 13.3536C13.2598 13.4473 13.1326 13.5 13 13.5H3C2.86739 13.5 2.74021 13.4473 2.64645 13.3536C2.55268 13.2598 2.5 13.1326 2.5 13V9.5" stroke="#E0E0E0" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Can you change stroke/etc to currentColor? 🙏🏻

Reply:
You missed one path's stroke 😆

Reply:
Ahhhh yes sorry. Updated.